data/presets/presets/leisure/marina.json

```diff
@@ -31,7 +35,9 @@
         "boat"
     ],
     "tags": {
-        "leisure": "marina"
+        "leisure": "marina",
+        "seamark:type": "harbour",
+        "seamark:harbour:category": "marina"
```

Comment:
These should go under `addTags` since otherwise this preset won't match any existing features that don't have them.